localization/replace/english/wc_replacements_l_english.yml

```diff
@@ -64,4 +64,7 @@
  huscarl_flavor:1 "#F $vrykul_group$ Huscarls are masters of using shield walls to block incoming volleys of arrows.#!"
  
  arab:1 "Western Human"
- african:1 "Southern Human"
\ No newline at end of file
+ african:1 "Southern Human"
+ 
+ scandinavian_adventurer_fp1_modifier:0 "Exile"
+ scandinavian_adventurer_fp1_modifier_desc:0 "This character fled the homeland with an army of rogues and adventurers, seeking to carve out a realm elsewhere."
```

Comment:
Change the wording:
'the homeland' to 'their homeland'

Reply:
```suggestion
 scandinavian_adventurer_fp1_modifier_desc:0 "This character fled their homeland with an army of rogues and adventurers, seeking to carve out a realm elsewhere."
```